.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: java
+sudo: false
+install: true
```

Comment:
@thboileau I've made several tests on this topic (as you can see on [all the commits I've made on my fork of RF](https://github.com/bellingard/restlet-framework-java/commits/master)), and actually never managed to make the Ant build pass - even w/o the SonarQube analysis. TravisCI is always killing the Ant build process for some reason - this is why I had deactivated the Ant build on my fork (see this [commit](https://github.com/bellingard/restlet-framework-java/commit/c4d1eab37b8e381acd1e7e854ef82bbc0863d185) in which I added it again to prepare the PR). I hope you'll be able to figure that out!